src/components/ui/DesktopNavBar.tsx

```diff
@@ -21,7 +21,7 @@ export default function DesktopNavBar ({ branding, search, navList }: DesktopApp
     >
       <div className='w-fit'>{branding}</div>
 
-      <div className='w-9/12 2xl:w-full'>
+      <div className='w-2/5 2xl:w-full'>
```

Comment:
<img width="1422" alt="Screenshot 2023-10-06 at 12 23 00 PM" src="https://github.com/OpenBeta/open-tacos/assets/3805254/703554c8-1c02-4e14-a4dd-026fc9086944">

Can you get rid off `2xl:w-full` so that the search box doesn't expand to 100% on very large screen? (see attachment)?  We're good to merge after this change. Thanks!